apps/mobile/mobile-app/components/impact/latam-impact-section.tsx

```
@@ -0,0 +1,324 @@
+	return {
+		screenWidth,
+		CARD_PADDING,
+		CARD_WIDTH,
+		isTablet,
+		isLandscape
+	}
+}
+
+type FeatureCardData = {
+	id: string
+	icon: any
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)